src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StandardSamUserDirectives.scala

```diff
@@ -49,7 +49,10 @@ trait StandardSamUserDirectives extends SamUserDirectives with LazyLogging with
   def asAdminServiceUser: Directive0 = requireOidcHeaders.flatMap { oidcHeaders =>
     Directives
       .mapInnerRoute { r =>
-        if (!adminConfig.serviceAccountAdmins.contains(oidcHeaders.email)) {
+        if (
+          !adminConfig.serviceAccountAdmins.contains(oidcHeaders.email) &&
+          !adminConfig.serviceAccountAdmins.contains(WorkbenchEmail(s"${oidcHeaders.externalId.map(_.value).toOption.orNull}@uami.terra.bio"))
+        ) {
           reject(AuthorizationFailedRejection)
         } else {
           logger.info(s"Handling request for service admin account: ${oidcHeaders.email}")
```

Comment:
I think this would be easier to follow in the affirmative, without the nots
```
        if (
          adminConfig.serviceAccountAdmins.contains(oidcHeaders.email) ||
          adminConfig.serviceAccountAdmins.contains(WorkbenchEmail(s"${oidcHeaders.externalId.map(_.value).toOption.orNull}@uami.terra.bio"))
        ) {
          // do the logic on starting on line 58
        } else {
          reject(AuthorizationFailedRejection)
```

Reply:
Inverted boolean logic for readability in [ea35970](https://github.com/broadinstitute/sam/pull/1466/commits/ea359701ed0994c4ab517a75dafe378ab117b05c)